sale_coupon_advanced/models/sale_coupon_program.py

```diff
@@ -28,18 +28,16 @@ class SaleCouponProgram(models.Model):
     )
 
     def _check_promo_code(self, order, coupon_code):
-
         order_count = self._get_order_count(order)
         if self.first_order_only and order_count:
             return {"error": _("Coupon can be used only for the first sale order!")}
-        max_order_number = self.first_n_customer_orders
+        max_order_number = self.next_n_customer_orders
         if max_order_number and order_count >= max_order_number:
             return {
                 "error": _(
                     "Coupon can be used only for the first {} sale order!"
```

Comment:
```suggestion
                    "Coupon can be used only for the {} times!"
```

Reply:
If we remove `sale order`, then `the` should be omitted.

Reply:
Or not, read different part..:)